dll/cpl/desk/lang/en-US.rc

```diff
@@ -6,7 +6,7 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 FONT 8, "MS Shell Dlg"
 BEGIN
     LTEXT "A theme is a background plus a set of sounds, icons, and other elements to help you personalize your computer with one click.", IDC_STATIC, 5, 5, 235, 30
-    LTEXT "Theme:", IDC_STATIC, 5, 42, 55, 10
+    LTEXT "Theme:", IDC_STATIC, 5, 42, 55, 10//FIXME: add accel
```

Comment:
@Doug-Lyons Do you have a Windows XP or 2003 in english so that we can check this...

Reply:
![XP](https://github.com/user-attachments/assets/329bb60d-7384-4c2f-ac9a-cf490f974ac9)

Reply:
Confirming with a full screen shot.

![W2K3SP2_Themes](https://github.com/user-attachments/assets/8f6b9a2c-91d1-4609-a6ce-37a009be152f)

Reply:
I will apply the capital &T where it is possible.

Reply:
Resolved.